src/manage_sql.c

```diff
@@ -32937,6 +32940,22 @@ create_target (const char* name, const char* asset_hosts_filter,
            new_target, snmp_credential, "0");
     }
 
+  if (krb5_credential)
+    {
+      gchar *type = credential_type (krb5_credential);
+      if (strcmp (type, "krb5"))
+        {
+          sql_rollback ();
```

Comment:
This should free `type`. The existing cases have the same, so could be a new PR.

Reply:
Fixed in [0e49e1e](https://github.com/greenbone/gvmd/pull/2332/commits/0e49e1ec1be168cb7aa179f28150dc0a93541d52)